docs/source/architecture/permissioning_requirement.rst

```diff
@@ -257,7 +257,7 @@ Allow all transaction signers to submit transactions
 |                    |   transaction signer.                                  |
 |                    | - A network operator can configure the                 |
 |                    |   whole Sawtooth network to accept transactions signed |
-|                    |   by any batch signer.                                 |
+|                    |   by any transaction signer.                           |
```

Comment:
There is an assumption that the batch containing this transaction is signed by a batcher that is allowed. This is applicable to current and next section.